templates/data-provider/claim-card.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Card, Button } from '@oacore/design'
+import { classNames } from '@oacore/design/lib/utils'
+
+import styles from './styles.module.css'
+
+const ClaimCard = ({ name }) => (
+  <Card className={classNames.use(styles.card, styles.claimCard)}>
+    <div>
+      <b>Are you {name}&apos;s manager?</b> Access insider analytics, issue
+      reports and manage access to documents from your repository in the CORE
+      Dashboard! 👇
```

Comment:
This text has been changed:

```suggestion
      <b>Do you manage {name}?</b> Access insider analytics, issue
      reports and manage access to outputs from your repository in
      the&nbsp;CORE&nbsp;Dashboard!&nbsp;
      <span aria-hidden="true" role="img">👇</span>
```

Also, I added [accessibility to the emoji](https://www.npmjs.com/package/a11y-react-emoji).